monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -254,20 +258,24 @@ log_info "Deploying ServiceMonitors and Prometheus rules"
 log_verbose "Deploying cluster ServiceMonitors"
 
 if [ "$TRACING_ENABLE" == "true" ]; then
-  TEMPO_CHART_VERSION=${TEMPO_CHART_VERSION:-1.5.0}
   log_info "Tracing enabled..."
-  # # Add the grafana helm chart repo
+  # Add the grafana helm chart repo
```

Comment:
Adding full air-gap support for Tempo is being handled in a separate work effort.